Protobuild.toml

```diff
@@ -5,11 +5,6 @@ version = "unstable"
 # example that selects the ctrd vanity binary.
 # generator = "gogoctrd"
 
-# Plugins allows one to specify one or more plugins for use in generation.
-#
-# The common example grpc is provided below.
-plugins = ["grpc"]
```

Comment:
The new Go code generator doesn't support plugins. We may need to deprecate the field.